variables.tf

```diff
@@ -5,7 +5,7 @@ variable "input_tags" {
 }
 
 variable "acm_domain_name" {
-  description = "Name of the certificate to be used in AWS ACM"
+  description = "Name of the certificate to be created in AWS ACM"
```

Comment:
Should this read as FQDN?

Reply:
In most cases, yes. But I've seen wild cards being used in the name in a lot of cases. I can update the comment by prepending it with "/FQDN."

Reply:
I would, it makes it more user-friendly.